### PR TITLE
Integration test with InaSAFE Headless

### PR DESCRIPTION
**Pull request by ismailsunni**
- Adding setup for inasafe-headless
- Adding setup for mounting data
- Update setup for running unit test for celery tasks.

### CODE REVIEW

**Comment by ismailsunni** [comment]
Hi @lucernae do you know why travis says failed? I am not sure what's wrong with it.
```
.............................
----------------------------------------------------------------------
Ran 29 tests in 2.644s
OK
Destroying test database for alias 'default'...
The command "coverage run manage.py test user_map" exited with 0.
Done. Your build exited with 1.
```

**Comment by ismailsunni** [comment]
Ah, it was a flake8.... Looking at the wrong place.
I also disabled the unit test for task since it needs to setup the inasafe-headless...

File: deployment/ansible/development/roles/pycharm/templates/workspace-configuration.xml.j2 [hunk 1]
```diff
@@ -251,9 +251,11 @@
...
       <option name="INTERPRETER_OPTIONS" value="" />
       <option name="PARENT_ENVS" value="true" />
-      <envs />
-      <option name="SDK_HOME" value="" />
-      <option name="WORKING_DIRECTORY" value="" />
+      <envs>
+        {% include 'run-manager-environment.xml.j2' %}
+      </envs>
+      <option name="SDK_HOME" value="ssh://root@{{ interpreters.inasafe_django.domain_alias }}:{{ interpreters.inasafe_django.ssh_port }}/usr/local/bin/python" />
+      <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$/django_project/" />
```
Comment (lucernae) [inline deployment/ansible/development/roles/pycharm/templates/workspace-configuration.xml.j2]: Nice one bruv. Although I'm not using nosetests.

File: django_project/realtime/tasks/headless/test_headless_task.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+custom_map_template = os.path.join(
+    dir_path, custom_map_template_basename + '.qpt'
+)
+
+OUTPUT_DIRECTORY = os.environ.get(
+    'INASAFE_OUTPUT_DIR', '/home/headless_test/data/result/')
+
+
+@unittest.skipIf(
+    os.environ.get('TRAVIS', False), 'Test is not ready for Travis')
```
Comment (lucernae) [inline django_project/realtime/tasks/headless/test_headless_task.py]: So, it is skipped for now? :p

Comment (ismailsunni) [reply django_project/realtime/tasks/headless/test_headless_task.py]: Yeah, we need to setup the inasafe-headless in the travis. I am not sure what's the best way to do it in travis.

**Pull request merged**